src/store/wallet.js

```diff
@@ -123,7 +126,9 @@ export default {
     },
     setClaimingInfo(state, { info }) {
       Object.assign(state, { claimingFileInfo: info })
-      console.log(state.claimingFileInfo)
+    },
+    setDisclaimers(state, { result }) {
+      state.disclaimers[result.signature] = result
```

Comment:
The keys of `disclaimers` should be the disclaimer name
```suggestion
    setDisclaimers(state, { result }) {
      state.disclaimers['disclaimer_1'] = result
```